components/script/dom/document.rs

```diff
@@ -4196,149 +4196,127 @@ impl DocumentMethods for Document {
     }
 
     // https://html.spec.whatwg.org/multipage/#dom-document-open
-    fn Open(&self, _type: Option<DOMString>, replace: DOMString) -> Fallible<DomRoot<Document>> {
+    fn Open(&self, _unused1: Option<DOMString>, _unused2: Option<DOMString>) -> Fallible<DomRoot<Document>> {
+        // Step 1. Throw InvalidStateError if the document is an XML document.
```

Comment:
I don't think paraphrasing the code in step comments is a good idea, but I'll let @jdm decide whether to keep them or not. I usually always remove them when I patch such code afterwards.

Reply:
I agree with nox. Let's only have `// Step 1` and `// Step 2` instead of the spec text as well.

Reply:
Agreed on the spec text, I removed it from the comments.